.github/workflows/docs.yaml

```diff
@@ -19,14 +19,15 @@ jobs:
 
       - name: Install dependencies
         run: |
+          pip install ghp-import
```

Comment:
Does it need a `python -m` prefix?

Reply:
If it runs, it doesn't. But it doesn't hurt to do that just in case.

Reply:
Updated